src/makeRobot.js

```diff
@@ -37,8 +37,71 @@
  *
  * @return {object}
  */
-function makeRobot(name, wheels, version) {
+ function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    x: 0,
+    y: 0,
+    get info() {
+      return (`name: ${
+        this.name
+      }, chip version: ${
+        this.version
+      }, wheels: ${
+        this.wheels
+      }`);
+    },
+    get location() {
```

Comment:
```suggestion
    },

    get location() {
```
Добавляй отступы между методами для читабельности